src/category_theory/monoidal/natural_transformation.lean

```diff
@@ -103,6 +103,15 @@ def hcomp {F G : lax_monoidal_functor C D} {H K : lax_monoidal_functor D E}
   end,
   ..(nat_trans.hcomp α.to_nat_trans β.to_nat_trans) }
 
+/-- The cartesian product of two monoidal natural transformations is monoidal. -/
+def prod {F G : lax_monoidal_functor C D} {H K : lax_monoidal_functor C E}
+  (α : monoidal_nat_trans F G) (β : monoidal_nat_trans H K) :
+  monoidal_nat_trans (F.prod' H) (G.prod' K) :=
```

Comment:
Shouldn't there be both a `prod` and a `prod'` version of this? (Sorry if I've missed something.)

Reply:
Good question.  I'm not sure I need any of this for the final result but it doesn't hurt to add it for completeness.